xbmc/cores/VideoPlayer/Edl.cpp

```diff
@@ -632,35 +634,35 @@ bool CEdl::ReadPvr(const CFileItem &fileItem)
 
 bool CEdl::AddCut(Cut& cut)
 {
-  if (cut.action != CUT && cut.action != MUTE && cut.action != COMM_BREAK)
+  if (cut.action != Action::CUT && cut.action != Action::MUTE && cut.action != Action::COMM_BREAK)
   {
-    CLog::Log(LOGERROR, "%s - Not a CUT, MUTE, or COMM_BREAK! [%s - %s], %d", __FUNCTION__,
+    CLog::Log(LOGERROR, "%s - Not a Action::CUT, Action::MUTE, or Action::COMM_BREAK! [%s - %s], %d", __FUNCTION__,
```

Comment:
an action, not a action

Reply:
Not added by me. I will fix it anyway. Thanks.